<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>745</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>745</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AI Verify</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>731</width>
      <height>541</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_monitor">
     <attribute name="title">
      <string>Monitor</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plain_text_monitor">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>721</width>
        <height>511</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_setting">
     <attribute name="title">
      <string>Setting</string>
     </attribute>
     <widget class="QGroupBox" name="group_info">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>331</width>
        <height>491</height>
       </rect>
      </property>
      <property name="title">
       <string>Server Information</string>
      </property>
      <widget class="QLabel" name="lb_ip_title">
       <property name="geometry">
        <rect>
         <x>45</x>
         <y>30</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>IP : </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lb_port_title">
       <property name="geometry">
        <rect>
         <x>45</x>
         <y>60</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Port : </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lb_pw_title">
       <property name="geometry">
        <rect>
         <x>45</x>
         <y>120</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>PW : </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lb_id_title">
       <property name="geometry">
        <rect>
         <x>45</x>
         <y>90</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID : </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lb_ip">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>28</y>
         <width>171</width>
         <height>18</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color : rgb(200, 200, 200)</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lb_port">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>58</y>
         <width>171</width>
         <height>18</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color : rgb(200, 200, 200)</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lb_id">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>88</y>
         <width>171</width>
         <height>18</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color : rgb(200, 200, 200)</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lb_pw">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>118</y>
         <width>171</width>
         <height>18</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color : rgb(200, 200, 200)</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="group_th">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>10</y>
        <width>331</width>
        <height>491</height>
       </rect>
      </property>
      <property name="title">
       <string>Inspect Threshold</string>
      </property>
      <widget class="QTableWidget" name="table_threshold">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>291</width>
         <height>192</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>20</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>20</number>
       </attribute>
       <column/>
       <column/>
      </widget>
      <widget class="QPushButton" name="push_btn_reset">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>460</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
      <widget class="QPushButton" name="push_btn_apply">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>460</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>745</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
